CMakeLists.txt

```diff
@@ -14,6 +14,7 @@
 cmake_minimum_required(VERSION 3.22.1)
 
 project(bitsandbytes LANGUAGES CXX)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Debug code left in? Or do we need this? It seems to not be supported by all targets

Reply:
I like this option but it doesn't need to be here, so I'll remove it.